apps/sms/test/marionette/lib/views/composer/view.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+/* global module */
+
+var ComposerAccessor = require('./accessors');
+
+function ComposerView(client) {
+  this.client = client;
+  this.accessors = new ComposerAccessor(client);
+}
+
+ComposerView.prototype = {
+  // TODO Move these constants to marionette
+  KEYS: {
+    backspace: '\ue003',
+    enter: '\ue007'
+  },
+
+  get recipients() {
+    var recipients = this.accessors.recipients;
+    return recipients.map(function(recipient) {
+      return recipient.text();
+    });
+  },
+
+  addNewRecipient: function(recipient) {
+    var recipientsList = this.accessors.recipientsList;
+    recipientsList.tap();
+    recipientsList.sendKeys(recipient);
+    recipientsList.sendKeys(this.KEYS.enter);
+  },
+
+  correctRecipients: function(recipients) {
```

Comment:
I don't really like the name of this method, I had no idea what it did exactly until I read entire method.

What is wrong with using ```clear``` and ```add``` in the test itself consequently? It doesn't look like this method will be reused a lot, or I'm missing something?